### PR TITLE
Update to 4.4.1

### PR DESCRIPTION
**Pull request by THS-on**
This seems to start. I needed to clear `~/.var/app/org.musescore.MuseScore/`, but did not investigate why.

Needs a bit more testing if everything works as expected. (Note that I'm not a MuseScore user that can fully evaluate that)

### CODE REVIEW

File: org.musescore.MuseScore.yaml [hunk 1]
```diff
@@ -90,6 +90,8 @@ modules:
       - -DMUE_BUILD_CRASHPAD_CLIENT=OFF
       # disable in-app updater
       - -DMUE_BUILD_UPDATE_MODULE=OFF
+      # See: https://github.com/musescore/MuseScore/commit/696279e362afe72db5e92f8a47aa64b3a0e86a86
+      - -D MUE_COMPILE_USE_SYSTEM_HARFBUZZ=ON
```
Comment (hfiguiere) [inline org.musescore.MuseScore.yaml:94]:
```suggestion
      - -DMUE_COMPILE_USE_SYSTEM_HARFBUZZ=ON
```

Just for consistency.

**Comment by hfiguiere** [comment]
The crash is 
```
/usr/include/c++/13.2.0/array:211: constexpr const std::array<_Tp, _Nm>::value_type& std::array<_Tp, _Nm>::operator[](size_type) const [with _Tp = mu::engraving::PropertyValue; long unsigned int _Nm = 1532; const_reference = const mu::engraving::PropertyValue&; size_type = long unsigned int]: Assertion '__n < this->size()' failed.
```

It's a known one they still haven't fixed. Probably because their QA process doesn't include building with assertions which detect genuine programming errors.

There are a few:

https://github.com/musescore/MuseScore/issues?q=is%3Aissue+is%3Aopen+mu%3A%3Aengraving%3A%3APropertyValue

But this is #123

**Comment by hfiguiere** [comment]
This also close #119

**Comment by hfiguiere** [comment]
re: crash. I started it a second time and it worked.

File: org.musescore.MuseScore.yaml (hunk 1 above)
Comment (hfiguiere) [reply org.musescore.MuseScore.yaml:94]: beside this, LGTM

Comment (THS-on) [reply org.musescore.MuseScore.yaml]: fixed

**Comment by hfiguiere** [comment]
Thank you so much!

**Pull request merged**